onnx/test/runtime_backend_test.py

```diff
@@ -0,0 +1,609 @@
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+# pylint: disable=C0415,R0912,R0913,R0914,R0915,W0640,W0703
+
+import os
+import unittest
+
+import numpy as np
+from numpy import object_ as dtype_object
+from numpy.testing import assert_almost_equal  # type: ignore
```

Comment:
Looks like `assert_allclose` is recommended: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html#numpy.testing.assert_almost_equal.

I remember these functions do broadcasting. Do we want to wrap them and check shape and dtype as well?